test/src/blockMonitor/blockProcessor.test.ts

```diff
@@ -45,42 +53,60 @@ const blocksByHash: { [key: string]: IBlockStub } = {
     c10: { number: 10, hash: "c10", parentHash: "c9" }
 };
 
+describe("BlockProcessorStore", () => {
+    let db: any;
+    let store: BlockProcessorStore;
+
+    beforeEach(async () => {
```

Comment:
Maybe one more test - "setLastestHeadNumber overwites current head number"

Reply:
Done in c24a745.